diff --git a/autogen.sh b/autogen.sh
index 250f80d..e1adab8 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -26,7 +26,7 @@ fi
 pkg-config --print-errors gobject-introspection-1.0
 if [ $? != 0 ]; then
 	echo "You probably need to install 'libgirepository1.0-dev'"
-	exit 1
+#	exit 1
 fi
 
 # README and INSTALL are required by automake, but may be deleted by clean
diff --git a/configure.ac b/configure.ac
index b99af91..4322178 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1300,9 +1300,6 @@ if $PKG_CONFIG --uninstalled $PANGO_PACKAGES; then
 else
 	gtk_save_LIBS="$LIBS"
         LIBS="$PANGO_LIBS $LIBS"
-        AC_TRY_LINK_FUNC(pango_context_new, :, AC_MSG_ERROR([
-*** Can't link to Pango. Pango is required to build
-*** GTK+. For more information see http://www.pango.org]))
         LIBS="$gtk_save_LIBS"
 fi
 
@@ -1624,8 +1621,8 @@ GLIB_GSETTINGS
 # GObject introspection
 ##################################################
 
-GOBJECT_INTROSPECTION_CHECK(introspection_required_version)
-
+#GOBJECT_INTROSPECTION_CHECK(introspection_required_version)
+AM_CONDITIONAL(HAVE_INTROSPECTION, false)
 ##################################################
 # colord module
 #################################################
