diff --git a/glib-2.0.pc.in b/glib-2.0.pc.in
index 275fc01..3f8754d 100644
--- a/glib-2.0.pc.in
+++ b/glib-2.0.pc.in
@@ -11,6 +11,6 @@ Name: GLib
 Description: C Utility Library
 Version: @VERSION@
 Requires.private: @PCRE_REQUIRES@
-Libs: -L${libdir} -lglib-2.0 @INTLLIBS@
+Libs: -L${libdir} -lglib-2.0 @INTLLIBS@ @ICONV_LIBS@
 Libs.private: @G_THREAD_LIBS@ @G_LIBS_EXTRA@ @PCRE_LIBS@ @INTLLIBS@ @ICONV_LIBS@ @CARBON_LIBS@ @COCOA_LIBS@
 Cflags: -I${includedir}/glib-2.0 -I${libdir}/glib-2.0/include @GLIB_EXTRA_CFLAGS@
