--- libiconv/Makefile.devel	2013-08-28 13:18:50.800584885 +0200
+++ libiconv/Makefile.devel	2013-08-28 13:19:02.089131379 +0200
@@ -4,10 +4,10 @@
 
 SHELL = /bin/sh
 MAKE = make
-AUTOCONF = autoconf-2.69
-AUTOHEADER = autoheader-2.69
-AUTOMAKE = automake-1.12
-ACLOCAL = aclocal-1.12
+AUTOCONF = autoconf
+AUTOHEADER = autoheader
+AUTOMAKE = automake
+ACLOCAL = aclocal
 GPERF = gperf
 CC = gcc
 CFLAGS = -O
--- libiconv/libcharset/Makefile.devel	2013-08-28 13:22:42.831038706 +0200
+++ libiconv/libcharset/Makefile.devel	2013-08-28 13:22:52.657025608 +0200
@@ -3,9 +3,9 @@
 
 SHELL = /bin/sh
 MAKE = make
-AUTOCONF = autoconf-2.69
-AUTOHEADER = autoheader-2.69
-ACLOCAL = aclocal-1.12
+AUTOCONF = autoconf
+AUTOHEADER = autoheader
+ACLOCAL = aclocal
 CP = cp
 RM = rm -f
 
--- libiconv/preload/Makefile.devel	2013-08-28 13:23:07.291565717 +0200
+++ libiconv/preload/Makefile.devel	2013-08-28 13:23:16.832057599 +0200
@@ -3,8 +3,8 @@
 
 SHELL = /bin/sh
 MAKE = make
-AUTOCONF = autoconf-2.69
-ACLOCAL = aclocal-1.12
+AUTOCONF = autoconf
+ACLOCAL = aclocal
 CP = cp
 RM = rm -f
 
